autoload/neomake.vim

```diff
@@ -512,7 +512,7 @@ function! s:AddExprCallback(maker) abort
             call setqflist(list, 'r')
         endif
     endif
-    if counts_changed
+    if counts_changed && has_key(a:maker, 'bufnr')
         call s:neomake_hook('NeomakeCountsChanged', {
                     \ 'file_mode': a:maker.file_mode,
                     \ 'bufnr': a:maker.bufnr})
```

Comment:
Sure!

For writing a test I wonder how to reproduce it.. you wrote:

> I also made a bugfix in neomake.vim which wasn't handling the
non-existance of bufnumber that could happen when running a maker
without having any files opened in neovim.